src/main/java/io/kestra/plugin/ai/domain/ChatConfiguration.java

```diff
@@ -17,46 +17,58 @@
+    )
     private Property<Integer> topK;
 
-    @Schema(title = "topP")
+    @Schema(
+        title = "Top-P (nucleus sampling)",
+        description = "Selects from the smallest set of tokens whose cumulative probability is ≤ topP. Typical values are 0.8–0.95. Lower values make the output more focused, higher values increase diversity."
+    )
     private Property<Double> topP;
 
-    @Schema(title = "seed")
+    @Schema(
+        title = "Seed",
+        description = "Optional random seed for reproducibility. Provide a positive integer (e.g., 42, 1234). Using the same seed with identical settings produces repeatable outputs."
+    )
     private Property<Integer> seed;
 
     @Schema(
-        title = "Whether to log LLM requests",
-        description = "Log will be sent to the server log in DEBUG."
+        title = "Log LLM requests",
+        description = "If true, prompts and configuration sent to the LLM will be logged at DEBUG level."
     )
     private Property<Boolean> logRequests;
 
     @Schema(
-        title = "Whether to log LLM responses",
-        description = "Log will be sent to the server log in DEBUG."
+        title = "Log LLM responses",
+        description = "If true, raw responses from the LLM will be logged at DEBUG level."
     )
     private Property<Boolean> logResponses;
 
     @Schema(
-        title = "The LLM response format, 'TEXT' by default",
+        title = "Response format",
         description = """
-            By default, LLM responses are non-structured texts.
-            You can ask for a different response format but be aware that not all LLM providers support setting the response format,
-            and the ones that support it may not support it in conjunction with tools.
-            If you specify a JSON schema, the name of the response will be `output`, you would be able to refer it in your prompt."""
+            Defines the expected output format. Default is plain text.
+            Some providers allow requesting JSON or schema-constrained outputs, but support varies and may be incompatible with tool use.
+            When using a JSON schema, the output will be returned under the key `output`, which can be referenced in prompts."""
     )
-    private ResponseFormat responseFormat;
+    private Property<ResponseFormat> responseFormat;
 
     public dev.langchain4j.model.chat.request.ResponseFormat computeResponseFormat(RunContext runContext) throws IllegalVariableEvaluationException {
         if (responseFormat == null) {
             return dev.langchain4j.model.chat.request.ResponseFormat.TEXT;
         }
 
-        return responseFormat.to(runContext);
+        // Use the nested ResponseFormat class which has the proper to() method
+        return runContext.render(responseFormat).as(ResponseFormat.class).orElse(ResponseFormat.builder().build()).to(runContext);
```

Comment:
All ResponseFormat properties are already dynamic so it's not really needed to set it as a dynamic property itself, it may behaves surprisingly.

Reply:
thanks, will remove and see if CI passes without it 👍